packages/protractor/protractor_web_test.bzl

```diff
@@ -261,6 +261,8 @@ def protractor_web_test(
         entry_point = Label(protractor_entry_point),
         data = srcs + deps + data + [Label(d) for d in peer_deps],
         testonly = 1,
+        # TODO: make protractor binary not depend on monkey-patched require()
```

Comment:
That's actual protractor and something we can't fix?

Reply:
pretty sure it's our wrapper code. I figured our protractor package isn't that popular so i didn't spend much time here digging in to figure out why it's not working, but i think this is the main one to come back to and try to fix by 3.0